tests/all_tests.js.regenerate.js

```diff
@@ -24,7 +24,8 @@ console.log('    } else {');
 console.log('      return exported_mains[`${fn}.js`];');
 console.log('    }');
 console.log('  } catch(e) {');
-console.log('    console.error("Error loading `./${fn}.js`, did you run `./scripts/gen_all_js.sh`?");');
+;
```

Comment:
:-/

Reply:
It didn't print the right thing before, apparently :-/